analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -383,8 +376,9 @@ private void updateAllExtraGradleFilesWithGmeRepos() throws IOException, Manipul
                 DirectoryFileFilter.DIRECTORY);
         for (File extraGradleScript : extraGradleScripts) {
             final List<String> lines = FileUtils.readLines(extraGradleScript, Charset.defaultCharset());
+            final String first = org.jboss.gm.common.utils.FileUtils.getFirstLine(lines);
 
-            if (!APPLY_GME_REPOS.equals(org.jboss.gm.common.utils.FileUtils.getFirstLine(lines))) {
+            if (!APPLY_GME_REPOS.equals(first)) {
```

Comment:
Why this change?

Reply:
I will revert it. Originally, I needed to pull it out to a variable for some other reason that's no longer in the code.

Reply:
Reverted.